apps/server/src/modules/tool/external-tool/controller/dto/request/external-tool-create.params.ts

```diff
@@ -58,6 +58,13 @@ export class ExternalToolCreateParams {
 	@ApiProperty()
 	isHidden!: boolean;
 
+	@IsBoolean()
+	@ApiProperty({
+		type: Boolean,
```

Comment:
Maybe some description would be useful. Something about that the tool is not launchable when it is deactivated